app/models/hets_instance.rb

```diff
@@ -1,10 +1,36 @@
 class HetsInstance < ActiveRecord::Base
+  class Error < ::StandardError; end
+  class NoRegisteredHetsInstanceError < Error
+    DEFAULT_MSG = 'There is no registered HetsInstance for this application'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.